scrunch/datasets.py

```diff
@@ -371,12 +383,23 @@ def __init__(self, obj, order, parent=None):
         # Load all the elements.
         for element in obj[self.name]:
             if isinstance(element, six.string_types):
-                _id = element.split('/')[-2]
-                var = self.order.vars.get(_id)
-                if var:
-                    self.elements[var['alias']] = var
+                if 'datasets' not in element or 'variables' in element:
```

Comment:
Can you elaborate a bit more what is happening here?

In a shoji:order we should just be able to match the urls of the mentioned resources to the associated catalog. This looks like it is special casing in the parts of the URL to behave distinctly.

Alternatively, if this is a situation of relative vs absolute URLs, I think that pycrunch has some helpers to conver urls to relative/absolute, We should address that matching of urls with pycrunch and not here,